depstability/records/depnames-6.1.2.txt

```diff
@@ -81,7 +81,7 @@ github.com/ncw/swift
 github.com/nu7hatch/gouuid
 github.com/nxadm/tail
 github.com/olekukonko/tablewriter
-github.com/onsi/ginkgo
+github.com/onsi/ginkgo/v2
```

Comment:
Should these depstability records be changed? I'm not familiar with depstability, but it feels like these should be historical records of the dependencies for specific versions. The changes to 7.11.0 and 7.12.0 seem especially strange.

Reply:
hmm. I'm not sure why the other versions got updated, let me dig more into this, all i did was run the test and looks like that updated the files

Reply:
fixed this issue and pushed the change